converters/google/fake_resource_data.go

```diff
@@ -77,6 +78,7 @@ func (d *FakeResourceData) HasChange(string) bool             { return false }
 func (d *FakeResourceData) Set(string, interface{}) error     { return nil }
 func (d *FakeResourceData) SetId(string)                      {}
 func (d *FakeResourceData) GetProviderMeta(interface{}) error { return nil }
+func (d *FakeResourceData) Timeout(key string) time.Duration  { return time.Duration(1) }
```

Comment:
required for compatibility with the latest versions of terraform-google-conversion